<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWDialog</class>
 <widget class="QDialog" name="QWDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>223</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog by Designer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="chkBoxUnder">
        <property name="text">
         <string>Underline</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkBoxItalic">
        <property name="text">
         <string>Italic</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkBoxBold">
        <property name="text">
         <string>Bold</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QRadioButton" name="rBtnBlack">
        <property name="text">
         <string>Black</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rBtnRed">
        <property name="text">
         <string>Red</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rBtnBlue">
        <property name="text">
         <string>Blue</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="txtEdit">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="plainText">
      <string>Hello, world
It is my demo.</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="text">
        <string>确 定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>取 消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>退 出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>QWDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>61</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>QWDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>QWDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
